fr.js

```diff
@@ -1,346 +1,351 @@
-    find_friends_heading: `Trouver des amis sur MyDoge`,
-    find_friends_description: `Autorisez l'accès aux contacts et facilitez la recherche d'amis et de contacts sur MyDoge.`,
-    find_friends_button: `Retrouver des amis`,
-    later_button: `je ferai ça plus tard`,
-    own_address_error: `Vous ne pouvez pas envoyer à votre propre adresse`,
-    invalid_address_error: `Adresse invalide`,
-    results_title: `Résultats`,
-    all: `Tous`,
-    pending: `En attente`,
-    enter_pin: `Entrez le code PIN`,
-    enter_pin_to_confirm: `Entrez votre code PIN de portefeuille pour confirmer`,
-    welcome_back: `Content de te revoir`,
-    enter_wallet_pin: `Entrez votre code PIN de portefeuille`,
-    forgot_pin: `Code PIN oublié`,
-    forgot_pin_title: `Code PIN oublié`,
-    forgot_pin_body: `Voulez-vous utiliser la biométrie pour déverrouiller votre portefeuille ?`,
-    share: `Partager`,
-    address_copied_to_clipboard: `Adresse copiée dans le presse-papiers`,
-    follows_you: `Vous suit`,
-    official_mydoge_profile: `Profil officiel MyDoge`,
-    official_mydoge_commerce_profile: `Profil officiel de MyDoge Commerce`,
-    official_foundation_profile: `Profil officiel de la Fondation`,
-    follow_back: `Suivre`,
-    follow: `Suivre`,
-    tip: `Conseil`,
-    share_profile: `Partager le profil`,
-    unfollow: `Ne plus suivre`,
-    unblock: `Débloquer`,
-    block_or_report: `Bloquer ou signaler`,
-    copy_profile_url: `Copier l'URL du profil`,
-    memes_error: `Erreur lors du chargement des mèmes. Veuillez revenir en arrière et réessayer.`,
-    user_profile_hidden: `Le profil utilisateur est masqué`,
-    error_loading_profile: `Erreur de chargement du profil. Veuillez revenir en arrière et réessayer.`,
-    uid_not_specified: `Erreur : UID non spécifié. Veuillez en informer le support.`,
-    block_or_report_description: `MyDoge examinera tous les profils signalés pour toute activité contraire à nos directives. Vous pouvez également bloquer le profil pour ne plus voir leurs publications.`,
-    nevermind: `Pas grave`,
-    report: `Rapport`,
-    block: `Bloc`,
-    unblock_user: `Débloquer l'utilisateur?`,
-    unblock_user_description: `Cela débloquera l'utilisateur et vous continuerez à voir ses publications.`,
-    you_have_no_posts_yet: `Vous n'avez pas encore de messages`,
-    no_posts_yet: `Pas encore de messages`,
-    amount: `Montant`,
-    custom: `Coutume`,
-    add_a_message: `Ajouter un message`,
-    find_friends: `Retrouver des amis`,
-    cancel: `Annuler`,
-    i_understand_continue: `Je comprends, continue`,
-    hey_are_you_sure: `Hé! Es-tu sûr?`,
-    delete_backup_confirmation: `Une fois que vous supprimez cette sauvegarde, vous perdez votre sauvegarde cloud, cette action ne peut pas être annulée`,
-    all_transactions: `toutes transactions`,
-    pending_transactions: `Transactions en attente`,
-    view_on: "Voir sur", // View on
-    invoice_id_copied: "Rechnungsnummer kopiert", // Invoice ID copied
```

Comment:
This appears to be german?

Reply:
Great catch. I've update the mistranslated keya